.github/workflows/subsplit.yml

```diff
@@ -23,5 +23,7 @@ jobs:
             - name: Install subsplit
               run: sudo cp ~/subsplit/git-subsplit.sh "$(git --exec-path)"/git-subsplit
 
+            - uses: nelonoel/branch-name@v1.0.1
+
             - name: Subsplit
-              run: bash build/tenancy-split.sh
+              run: bash build/tenancy-split.sh ${BRANCH_NAME}
```

Comment:
Can't you simply use `GITHUB_REF` for this? 🤔 https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables

Reply:
Nope, GITHUB_REF holds the full ref, not just the branch name.  We need the actual branch name